src/components/Banner/Banner.tsx

```diff
@@ -170,6 +202,8 @@ export class Banner extends React.PureComponent<BannerProps, never> {
                 role={ariaRoleType}
                 aria-live={stopAnnouncements ? 'off' : 'polite'}
                 onMouseUp={handleMouseUp}
+                onKeyUp={handleKeyUp}
+                onBlur={handleBlur}
```

Comment:
I'm curious what others think of this. The current iteration of banner just uses a transition delay to somewhat prevent the focus ring from showing on click. The result isn't too bad, only really noticeable when you highlight text or on a click and hold.

https://screenshot.click/screencast_2020-01-29_13-28-10.mp4

I wonder if the re-renders added here are necessary

Reply:
I think this re-render is pretty cheap. A transition delay could lead to confusion imo.

Reply:
It is pretty cheap, and if we're doing it on other elements then it probably makes sense to do it on banner as well. I doubt too many people are clicking banners anyway